### ISSUE TITLE
[layoutlmv3]: Issue with label format? Inference yields boundary boxes that are too short.

### ISSUE THREAD

**Issue by OGiesecke** [microsoft/unilm#924]
Hi, 

I am working on object detection with layoutlmv3.

I am using the publaynet fine tuned model and have a training set with about 600 documents. 

The issue that I am facing is that the predicted boundary boxes are only kind of correct. In most of the documents the predicted boundary are "too short". Meaning that the lower y coordinate is usually too small.

As an example I have attached you an example from my evaluation dataset. It is the case in almost every single inference picture. Thus, I am trying to get some ideas to troubleshoot.

I double checked that the drawn boundary boxes in the inference is correctly done.

![ak_state_of_alaska_2020_p144](https://user-images.githubusercontent.com/40527435/202876949-c6e41860-ee7d-4b2f-b34b-c2d7ef7cdd4a.jpeg)

![inference](https://user-images.githubusercontent.com/40527435/202876954-87cab2cc-5f26-4142-a661-d9e7abfb9931.jpeg)

Any ideas would be greatly appreciated.

**Comment by HYPJUDY** [comment]
Hi. My idea is to try to train a model that overfits your training set, and then evaluate examples from the training set to see if they have the same issue. If so, there may be some deviations in the data (e.g., labels) or image processing (e.g., resizing/cropping). Otherwise, the gap between the training/test sets may be due to insufficient training.

**Comment by OGiesecke** [comment]
Hi! That was super helpful. I was able to identify the problem in the training data. In the end it turned out to be a tiny issue with the label format. The model performs very well on this dataset. Thank you again for your help!

**Comment by ashishupadhyaa** [comment]
Can you please tell me the location of the inference file?
